content/api/commands/origin.md

```diff
@@ -108,8 +108,9 @@ cy.get('h1').contains('My cool site under test')
 **<Icon name="angle-right"></Icon> url** **_(String)_**
 
 A URL specifying the secondary origin in which the callback is to be executed.
-This should at the very least contain a hostname, and may also include the
-protocol, port number & path. Query params are not supported.
+This should contain the full hostname, which includes the subdomain (if
+applicable), domain, and top-level domain. This may also include the protocol,
+port number & path. Query params are not supported.
```

Comment:
Do we have examples of what these parts of a URL equate to in the cypress docs at all? Newbies might know know what these parts are

Reply:
I think the closest thing that we own is probably the [cy.location()](https://docs.cypress.io/api/commands/location) command that shows each of these parts, or at least what hostname is?

Reply:
Maybe we can add something similar to [Node's URL doc](https://nodejs.org/api/url.html#url-strings-and-url-objects)?

Reply:
or even link to Node's docs to learn more?

Reply:
Looks like we have [done that before](https://github.com/cypress-io/cypress-documentation/blob/fd6448e0a1183c3b6e88a09b86510dd0320ba851/content/guides/references/configuration.md?plain=1#L492-L493).

Reply:
I don't think we have historically linked out, but I wouldn't be opposed to that or including a table like the Node docs have?

Reply:
Going to come up with a table for us to put in our docs, just not sure where yet. Going to move this to draft in the meantime.

Reply:
How do we feel about something like this?
```
┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                     href                                                    │
├──────────┬──┬─────────────────────┬─────────────────────────────────────┬───────────────────────────────────┤
│ protocol │  │        auth         │                  host               │           path            │ hash  │
│          │  │                     ├─────────────────────────────────────┼───────────────────────────┤       │
│          │  │                     │             hostname         │ port │ pathname │     search     │       │
│          │  │                     │ subdomain │ superdomain(sd)  │      │          │                │       │
│          │  │                     │           │  domain │  TLD   │      │          ├─┬──────────────┤       │
│          │  │                     │           │         │        │      │          │ │    query     │       │
"  https:   //    user   :   pass   @    sub    . example .  com   : 8080   /p/a/t/h  ?  query=string   #hash "
│          │  │          │          │             hostname         │ port │          │                │       │
│          │  │          │          ├──────────────────────────────┴──────┤          │                │       │
│ protocol │  │ username │ password │                 host                │          │                │       │
├──────────┴──┼──────────┴──────────┼─────────────────────────────────────┤          │                │       │
│   origin    │                     │                origin               │          │                │       │
├─────────────┼─────────────────────┤───────────┬─────────────────────────┤          │                │       │
│ (sd) origin │                     │           │   (sd) origin           │          │                │       │
├─────────────┴─────────────────────┴───────────┴─────────────────────────┴──────────┴────────────────┴───────┤
│                                                       href                                                  │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

Reply:
Looks great. I took a stab at simplifying it a bit. I got rid of some duplication (that exists in the original) and the auth part, which I think we can do without for our purposes.

```
┌───────────────────────────────────────────────────────────────────────────────────────┐
│                                         href                                          │
├──────────┬──┬─────────────────────────────────────┬───────────────────────────┬───────┤
│ protocol │  │               host                  │           path            │ hash  │
│          │  ├──────────────────────────────┬──────┼──────────┬────────────────┤       │
│          │  │           hostname           │ port │ pathname │     search     │       │
|          |  ├───────────┬──────────────────┤      │          │                │       │
│          │  │ subdomain │ superdomain (sd) │      │          │                │       │
|          |  ├───────────┼─────────┬────────┤      │          ├─┬──────────────┤       │
│          │  │           │  domain │  TLD   │      │          │ │    query     │       │
"  https:   //     sub    . example .  com   : 8080   /p/a/t/h  ?  query=string   #hash "
├──────────┴──┴───────────┴─────────┴────────┴──────┤          │                │       │
│                      origin                       │          |                │       │
├─────────────┬───────────┬─────────────────────────┤          │                │       │
│ (sd) origin │           │   (sd) origin           │          │                │       │
└─────────────┴───────────┴─────────────────────────┴──────────┴────────────────┴───────┘
```

Reply:
I like it! It's easier to digest imo. Just need to find a spot to put it. Gonna play around with docs in development and see what makes sense. Any suggestions?